pyproject.toml

```diff
@@ -85,6 +85,9 @@ sfno = [
     "tensorly", # makani
     "tensorly-torch" # makani
 ]
+aurora = [
+    "microsoft-aurora@git+https://github.com/ivanauyeung/aurora@timm_update",
```

Comment:
I see your referencing your fork here, just curious what was changed?

Reply:
It's basically [2 lines in the aurora source](https://github.com/microsoft/aurora/compare/main...ivanauyeung:aurora:timm_update#diff-eacdde5099ff6d4476759c67420a5804816c95b326761ce7b628f0501f72a3b7L9) that are dependent on an older `timm` version than what is provided by the physicsnemo container. Since e2studio doesn't technically have requirements on `timm` version, we could just add another dep here in the `toml` under `aurora` and use their source rather than Ivan's fork.

Reply:
My worry with the fixed timm version is that it could conflict with other models / packages... I think for testing purposes we can use the fork as the install method with the fixes, but them keep the package in the toml.

This is a larger issue @swbg brought up with the package management

Created an issue to track this in the mean time:
https://github.com/NVIDIA/earth2studio/issues/235